integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ItMiiSampleHelper.java

```diff
@@ -0,0 +1,376 @@
+      String dbImageName = (KIND_REPO != null
+          ? KIND_REPO + DB_IMAGE_NAME.substring(BASE_IMAGES_REPO.length() + 1) : DB_IMAGE_NAME);
+      String jrfBaseImageName = (KIND_REPO != null
+          ? KIND_REPO + FMWINFRA_IMAGE_NAME.substring(BASE_IMAGES_REPO.length() + 1) : FMWINFRA_IMAGE_NAME);
+      String dbNamespace = envMap.get("dbNamespace");
+
+      envMap.put("DB_IMAGE_NAME", dbImageName);
+      envMap.put("DB_IMAGE_TAG", DB_IMAGE_TAG);
+      envMap.put("DB_NODE_PORT", "none");
+      envMap.put("BASE_IMAGE_NAME", jrfBaseImageName);
+      envMap.put("BASE_IMAGE_TAG", FMWINFRA_IMAGE_TAG);
+      envMap.put("POD_WAIT_TIMEOUT_SECS", "1000"); // JRF pod waits on slow machines, can take at least 650 seconds
+      envMap.put("DB_NAMESPACE", dbNamespace);
+      envMap.put("DB_IMAGE_PULL_SECRET", BASE_IMAGES_REPO_SECRET); //ocr/ocir secret
+      envMap.put("INTROSPECTOR_DEADLINE_SECONDS", "600"); // introspector needs more time for JRF
+
+      // run JRF use cases irrespective of WLS use cases fail/pass
+      previousTestSuccessful = true;
+      execTestScriptAndAssertSuccess(domainTypeName, "-db,-rcu", "DB/RCU creation failed");
+    }
+
+    execTestScriptAndAssertSuccess(
+        domainTypeName,
+        "-initial-image,-check-image-and-push,-initial-main",
+        "Initial use case failed"
+    );
+  }
+
+  /**
+   * Test MII sample WLS or JRF update1 use case.
+   */
+  public static void callUpdateUseCase(String args,
+                                       String errString) {
+    if (args.contains("update3")) {
+      String imageName = (domainTypeName.equals(DomainType.WLS))
+          ? MII_SAMPLE_WLS_IMAGE_NAME_V2 : MII_SAMPLE_JRF_IMAGE_NAME_V2;
+      envMap.put("MODEL_IMAGE_NAME", imageName);
+    }
+
+    execTestScriptAndAssertSuccess(domainTypeName, args, errString);
+  }
+
+  /**
+   * Delete DB deployment for FMW test cases and Uninstall traefik.
+   */
+  public static void tearDownAll() {
+    logger = getLogger();
+    // uninstall traefik
+    if (traefikNamespace != null) {
+      logger.info("Uninstall traefik");
+      Command.withParams(new CommandParams()
+          .command("helm uninstall traefik-operator -n " + traefikNamespace)
+          .redirect(true)).execute();
+    }
+
+    // db cleanup or deletion
+    if (domainTypeName.equals(DomainType.JRF) && envMap != null) {
+      logger.info("Running samples DB cleanup");
+      Command.withParams(new CommandParams()
+          .command(MII_SAMPLES_SCRIPT + " -precleandb")
+          .env(envMap)
+          .redirect(true)).execute();
+    }
+  }
+
+
+  /**
+   * Set env variables map.
+   * @param newEnvMap a map contains env variables
+   */
+  public static void setEnvMap(Map<String, String> newEnvMap) {
+    ItMiiSampleHelper.envMap = newEnvMap;
+  }
+
+  /**
+   * Set domain type.
+   * @param domainTypeName domain type name
+   */
+  public static void setDomainType(DomainType domainTypeName) {
```

Comment:
minor: this method can be removed - domainType can be a parameter of initAll

Reply:
removed